tests/test_unittest.py

```diff
@@ -0,0 +1,26 @@
+"""Test unittest integration."""
+# pylint: disable=missing-docstring,no-self-use
+import sys
+import unittest
+
+from flexmock import flexmock
+
+
+class TestUnitTestIntegration(unittest.TestCase):
+    """Flexmock unittest integration specific tests."""
+
+    def test_failed_test_case(self):
+        """This tests that after a successful tests, failing flexmock assertions
+        will change the test result from successful to failed.
+        """
+        flexmock().should_receive("this_test_should_fail").once()
+
```

Comment:
should there be also the teardown test with `ordered` ?

Reply:
I think it's not needed since we run the whole test suite with unittest and there are multiple tests that fail if teardown doesn't work.